src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -34,12 +35,14 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
         commit(ModelCalibrateMutation.SetOptionsData, options);
         commit(ModelCalibrateMutation.Calibrating);
         const response = await api<ModelRunMutation, ModelCalibrateMutation>(context)
-            .withSuccess(ModelRunMutation.RunResultFetched)
+            .ignoreSuccess()
             .withError(ModelCalibrateMutation.SetError)
             .freezeResponse()
             .postAndReturn<ModelResultResponse>(`/model/calibrate/${modelRunId}`, {options, version});
 
         if (response) {
+            const data = freezer.deepFreeze(response.data);
+            commit({type: `modelRun/${ModelRunMutation.RunResultFetched}`, payload: data}, {root: true});
```

Comment:
I've ended up committing the main success route here, as this was failing going through the apiService withSuccess, which I don't think really supports mutation non-root modules which aren't the home module of the commit (because it doesn't construct the path for them)..?